tests/jballerina-unit-test/src/test/resources/test-src/isolated-objects/isolated_objects.bal

```diff
@@ -41,7 +41,6 @@ final readonly & string[] immutableStringArray = ["hello", "world"];
 
 type IsolatedObjectType isolated object {
     int a;
-    string[] b;
 };
 
 isolated class IsolatedClassOverridingMutableFieldsInIncludedIsolatedObject {
```

Comment:
With the above change, this isn't quite overriding, right? Shall we change the field of `a` in this class to byte maybe?